src/test/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMessageHandlerTest.java

```diff
@@ -66,62 +64,48 @@ void handleInboundMeshMessageNoTransactionsDoesNotPublishToGpOutboundQueue() {
         inboundMessageHandler.handle(meshMessage);
 
         verify(edifactParser, times(2)).parse(any());
-        verify(edifactToFhirService, never()).convertToFhir(any(Transaction.class));
```

Comment:
I would say this is still relevant. Instead we can verify the `convertToFhir()` is never called with a `Message` being passed

Reply:
Yep, it'd definitely still relevant 

Have reintroduced this again for `Message.class` in 697ebb8